### PR TITLE
111, initial commit, JENKINS_OPTS put into params/jenkins/build file,…

### LINKED ISSUE #111: Increase SessionTimeout for Jenkins
From @mcanoy 

```
oc set env dc/jenkins --list -n kevin
# deploymentconfigs jenkins, container jenkins
JENKINS_OPTS=--sessionTimeout=720
```

### PR DESCRIPTION
**Pull request by JoshuaTPritchett**
resolves #111

### CODE REVIEW

**Comment by sherl0cks** [comment]
@JoshuaTPritchett first, I updated your comment. The `resolves` will close the issue automatically if this PR is merged

as to the content of the PR, this one is a bit trickier because we need to modify the `DeployConfig`, not the `BuildConfig`, and currently this repo uses the out of the box openshift deployment template. We recently agreed to move to a custom jenkins deploy template, so this work could be very helpful.

basic what you need to do is to add a template based on `oc get template jenkins-ephemeral -n openshift -o yaml`. Make sure the template is `yaml`, not `json`. Then you can add your env to new template. Make that change, and we can go from there.

**Comment by JoshuaTPritchett** [comment]
Just a couple quick notes:
1) When using Ansible on Ubuntu 14.04 it must be updated otherwise casl-ansible will not work (Ansible 2.1 or something crazy!)
2) Consider adding examples to the README for tagging with the openshift-applier roles for casl instead of just hard referencing docs

**Comment by sherl0cks** [comment]
@JoshuaTPritchett we tend to stay on the bleeding edge of ansible. I'm testing with `2.4.2.0` right now. We're in process of providing a docker container to actually do the playbook runs so this is a bit easier. 

https://github.com/rht-labs/labs-ci-cd/issues/113 update RE: docs

**Comment by JoshuaTPritchett** [comment]
It's cool, I tested with casl-ansible. I believe my changes are going to be good to go, let me know and I'll finalize this today.

**Comment by sherl0cks** [comment]
@JoshuaTPritchett looks like Jenkins ephemeral and the new persistent entry are using the same params file, which is messing up the CI job because the params file is modified twice.

I need to check in with my team around moving to jenkins peristent, now may be a good time

**Comment by sherl0cks** [comment]
@JoshuaTPritchett I checked in with the team. let's move this to the persistent template, so please remove the ephemeral template completely from the inventory and template inventory. then we can rerun the CI and go from there

**Comment by JoshuaTPritchett** [comment]
@sherl0cks sounds, good, I put both to see which you would prefer. Would it be better to remove, or leave in case someone else wants to use it, but just remove from inventory?

**Comment by sherl0cks** [comment]
@JoshuaTPritchett remove. in general, we want this inventory to be prescriptive.

**Comment by JoshuaTPritchett** [comment]
Done.

**Comment by JoshuaTPritchett** [comment]
@sherl0cks What's the status on this? Is there something I missed?

File: openshift-templates/jenkins/jenkins-persistent-template.yaml [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+    replicas: 1
+    selector:
+      name: ${JENKINS_SERVICE_NAME}
+    strategy:
+      type: Recreate
+    template:
+      metadata:
+        creationTimestamp: null
+        labels:
+          name: ${JENKINS_SERVICE_NAME}
```
[the remark this answers is not in the document]
Comment (sherl0cks) [reply openshift-templates/jenkins/jenkins-persistent-template.yaml]: I'm going to make this change myself, so I can integrate a few other pieces

**Comment by sherl0cks** [comment]
@JoshuaTPritchett good work.

**Pull request merged**